terms.html

```diff
@@ -51,9 +51,12 @@
   </dd>
   <dt><dfn data-lt="entities|entity's">entity</dfn></dt>
   <dd>
-A thing with distinct and independent existence, such as a person,
-organization, or device that performs one or more roles in the ecosystem.
-  </dd>
+Anything that can be referenced in statements as an abstract or concrete noun.
+Entities include but are not limited to people, organizations, physical things, documents,
+abstract concepts, numbers, and strings. Any entity might perform roles
```

Comment:
While it's true that "numbers and strings" are examples of entities... I wonder if this is too meta for most readers of the document. I was trying to think of a "string" as a credential subject" and was struggling to think of anything other than a dictionary or thesaurus that would use that concept... and it's even harder to imagine a VC issued for a string or a number. 

Please consider removing the two examples and focusing more on entities that you'd expect to see a VC issued for (rather than more rare cases).